fares-data-build-tool/fdbt-site
# Feature/tfn 641 multiple service and number of products cookies

# Description

-   This PR removes the unused MULTIPLE_SERVICE_COOKIE and moves the NUMBER_OF_PRODUCTS_COOKIE to sessions.

# Testing instructions

-   Pull down this branch and run the site locally. Validate that the pages modified in this PR still work as expected. Also validate that the matching json for each of a single period product and multiple period products is of the correct form.

# Type of change


# Checklist:
